.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      args: [--safe]
+      language_version: python3.6
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v1.2.3
+    hooks:
+    -   id: flake8
+        args: [--max-line-length=88]
```

Comment:
we need to sync the pre-commit config with PXM's: https://github.com/mapbox/pxm/blob/dev/.pre-commit-config.yaml